src/main/java/edu/cornell/kfs/gl/CuGeneralLedgerConstants.java

```diff
@@ -6,6 +6,8 @@
 
 	public static final String ANNUAL_CLOSING_CHARTS_PARAM = "ANNUAL_CLOSING_CHARTS";
 	public static final String COLLECTOR_HEADER_RECORD_TYPE = "HD";
+	public static final String COLLECTOR_TRAILER_RECORD_TYPE = "TL";
+	public static final String COLLECTOR_FILE_DATE_FORMAT = "yyyy-MM-dd";
```

Comment:
If these constants are specific to the Concur processing, they should go into the ConcurConstants and be sub-classed or named as being Collector specific.